nss-day-cohort-45/Hello-World-The-Animation-Durations
# gt-auto

## Changes Made:
1) Automated cities, citizens, and landmark lists with JS
2) Altered nav bar styling

## How to Check:
1) Verify that all JS files in thailand directory have functioning code
2) Verify that nav bar style changes exist
